shim/dist/deb/debian/postinst

```diff
@@ -4,13 +4,20 @@ SHIM_CONFIG_DIR=/etc/inclavare-containers
 
 mkdir -p $SHIM_CONFIG_DIR
 cat << EOF > $SHIM_CONFIG_DIR/config.toml
-log_level = "info" # "debug" "info" "warn" "error"
+log_level = "debug" # "debug" "info" "warn" "error"
```

Comment:
Are you sure debug is now becoming the default?

Reply:
> Are you sure debug is now becoming the default?

Yes, Please see [shim readme.md](https://github.com/alibaba/inclavare-containers/tree/master/shim#step-2-configuration)